invenio_oauthclient/utils.py

```diff
@@ -117,7 +117,8 @@ def oauth_link_external_id(user, external_id=None):
                 id_user=user.id
             ))
     except IntegrityError:
-        raise AlreadyLinkedError(user, external_id)
+        if external_id['method'] is not 'cern':
```

Comment:
Please try to fix the problem which is [here](https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/utils.py#L76-L77):

```python
if account_info.get('user', {}).get('email'):
    return User.query.filter_by(email=account_info['user']['email']).first()
```